### ISSUE TITLE
missing htmlTag

### ISSUE THREAD

**Issue by r0b-** [romanschejbal/gassetic#68]
Hi,

after updating to v1.0.7 I got "missing htmlTag" errors for all my mimetypes. 
Shouldn't there a default tag like beofre?

Thanks!

**Comment by r0b-** [comment]
Ok I see - it's mandatory now

**Comment by r0b-** [comment]
I think it would make sense to add default script and style tags or is there a reason to avoid that?

**Comment by romanschejbal** [comment]
I've added some defaults for css and js. 😉

**Comment by r0b-** [comment]
Thanks a lot for the quick feedback, but I still have the same error - is the fix included in 1.0.8?

in my gassetic.dump.dev.yml:
```
- mimetype: js
  htmlTag: <!-- missing htmlTag js -> dev -->
  files:
    app:
      - /app.module.js
      ...
```

**Comment by romanschejbal** [comment]
I forgot about the dump file, sorry.. Please find the fix in 1.0.9

**Comment by r0b-** [comment]
js and css is working now, but I have two sections with the 'missing htmlTag' error:

- First one is optimizing & copying images
- Second one is combining html files (angular templates) into a single js file (angular templatecache) that needs to be inserted with js tag
